src/app/pages/component-sidenav/component-sidenav.spec.ts

```diff
@@ -15,17 +16,17 @@ describe('ComponentSidenav', () => {
 
   beforeEach(() => {
     fixture = TestBed.createComponent(ComponentSidenav);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
   });
 
   it('should close the sidenav on init', () => {
-    const component = fixture.componentInstance;
-
     // Spy on window.mediaMatch and return stub
     spyOn(window, 'matchMedia').and.returnValue({
       matches: true
     } as any);
 
-    fixture.detectChanges();
+    // TODO refactor this as none of these expectations are ever verified
```

Comment:
What do you mean by "verified"?

Reply:
Currently this test generates the following warning in the console:
```
'Spec 'ComponentSidenav should close the sidenav on init' has no expectations.'
```
I debugged it quite a bit, but I wasn't able to figure out how to re-write the test in a way that
1. Actually ran the `expect()` calls
1. Was configured in a way that `component.sidenav` was defined